docs/source/jupyter.rst

```diff
@@ -27,11 +27,11 @@ Jupyter notebooks
    :maxdepth: 1
    :caption: symbolic
    
-   /examples/Einstein Tensor symbolic calculation.ipynb
-   /examples/Lambdify symbolic calculation.ipynb
-   /examples/Playing with Contravariant and Covariant Indices in Tensors(Symbolic).ipynb
-   /examples/Predefined Metrics in Symbolic Module.ipynb
-   /examples/Ricci Tensor and Scalar Curvature symbolic calculation.ipynb
-   /examples/Symbolically Understanding Christoffel Symbol and Riemann Curvature Tensor using EinsteinPy.ipynb
-   /examples/Weyl Tensor symbolic calculation.ipynb
+   /examples/Symbolic Module - Einstein Tensor.ipynb
+   /examples/Symbolic Module - Lambdify.ipynb
+   /examples/Symbolic Module - Tensor Indexing.ipynb
+   /examples/Symbolic Module - Predefined Metrics.ipynb
+   /examples/Symbolic Module - Ricci Tensor and Scalar Curvature.ipynb
+   /examples/Symbolic Module - Christoffel Symbols and Riemann Curvature Tensor.ipynb
+   /examples/Symbolic Module - Weyl Tensor.ipynb
    
```

Comment:
@prakashaditya369 Leave only 1 blank line at the end.

Reply:
@JeS24  I think there is only 1 blank line at the end.

Reply:
![image](https://user-images.githubusercontent.com/31299799/105016512-bd106900-5a68-11eb-9c16-f5ab4a3f21a1.png)

Reply:
Actually this isn't showing in my text editor, so I updated it from github only.

Reply:
> Actually this isn't showing in my text editor, so I updated it from github only.
 Use a "code" editor. Perhaps, your text editor is interpreting that last blank line as EOF.